index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+
+    <link rel="stylesheet" href="index.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Profile</title>
+</head>
+
+<body>
+    <header>
+        <nav id="nav">
+            <div class="nav">
+                <a class="nav__logo" href="#">My Logo</a>
+    
+                <img class="nav__hambuger" src="images/hambuger.svg" alt="Icon">
+    
+            </div>
+        </nav>
+    </header>
+    
+    <section id="hero">
+        <div class="hero">
+            <h1 class="hero__heading">
+                I’m Najash
+                Glad to see you!
```

Comment:
- [Optional] Consider using an indentation of two spaces for readabilty of your code as this is [best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#html--css-best-practices) and convention.